### PR TITLE
Enumerable: Feature/add enumerable methdos

### PR DESCRIPTION
**Pull request by mateo951**
- Added readme file
- Added linters
- Added class, all and any enumerable methods, linters
- Added filter method

### CODE REVIEW

File: mylist.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
```
Comment (henatan99) [inline mylist.rb:7]: - [ ] You did an amazing job and your enumerable methods are working very fine and pass all the test cases given. But it seems you missed one project requirement. In MyList you are required to implement a method `each` that yields successive members of `@list`. Kindly make changes. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @mateo951 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mylist.rb (hunk 1 above)
Comment (mateo951) [reply mylist.rb:7]: Hey @henatan99,  thanks for your feedback! I dont quite understand that part. yields successive members... Should I create a method that prints out the list members with list.each method?

Comment (mateo951) [reply mylist.rb:7]: def each
      @list.each {
        |n| puts n 
      }
    end

Is something like this is what the description requires?

Comment (henatan99) [reply mylist.rb:7]: Kind of, but better to use `yield` method

Comment (henatan99) [reply mylist.rb:7]: Ok, nice to make changes even though it is not exactly as I expected. It is not explicitly stated in the requirements how the `each` method behaves.

**Review by henatan99 — APPROVED** [review]
Hi @mateo951 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**